### PR TITLE
changes for failure criteria

### PR DESCRIPTION
**Pull request by vinitsuri**
- Fixed indentations in all CLT Build task PS scripts
- Changes to add failure criteria of Avg.Response time by fetching the avg response time from the Result Summary controller

### CODE REVIEW

File: Tasks/QuickPerfTest/CltTasksUtility.ps1 [hunk 1]
```diff
@@ -189,3 +189,10 @@ function UploadSummaryMdReport($summaryMdPath)
 	}
 }
 
+function isNumericValue ($str) {
```
Comment (dpksinghal) [inline Tasks/QuickPerfTest/CltTasksUtility.ps1]: >isNumericValue [](start = 9, length = 14)

Minor: Capital 'I' plz

File: Tasks/QuickPerfTest/CltThresholdValidationHelper.ps1 [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+function ValidateAvgResponseTimeThreshold($cltAccountUrl,$headers,$testRunId,$avgResponseTimeThreshold)
+{
+	$resultsSummary = GetResultsSummary  $cltAccountUrl  $headers $testRunId
+	if($resultsSummary -and $resultsSummary.overallRequestSummary -and $resultsSummary.overallRequestSummary.averageResponseTime)
+	{
+		if($resultsSummary.overallRequestSummary.averageResponseTime -gt ($avgResponseTimeThreshold/1000))
+		{
+			return $false
+		}
+		return $true
```
Comment (dpksinghal) [inline Tasks/QuickPerfTest/CltThresholdValidationHelper.ps1]: minor: you can simply return the if clause with condition inverted.

File: Tasks/QuickPerfTest/Invoke-QuickPerfTest.ps1 [hunk 3]
```diff
@@ -109,30 +117,54 @@ $drop = CreateTestDrop $headers $dropjson $CltAccountUrl
...
+	}
+	else
+	{
+		$thresholdMessage="No thresholds violated."
+		$thresholdImage="bowtie-status-success"
+	}
+	$summary = ('[Test Run: {0}]({1}) using {2}.<br/>' -f  $run.runNumber, $webResultsUrl ,$run.name)
+	$summary = ('<span class="bowtie-icon {3}" />   {4}<br/>[Test Run: {0}]({1}) using {2}.<br/>' -f  $run.runNumber, $webResultsUri , $run.name, $thresholdImage, $thresholdMessage)
 
 	('<p>{0}</p>' -f $summary) >>  $summaryFile
```
Comment (dpksinghal) [inline Tasks/QuickPerfTest/Invoke-QuickPerfTest.ps1]: >('<p>{0}</p>' -f $summary) >>  $summaryFile [](start = 1, length = 43)

Anjani had changed this code to write in the ascii format. Am I missing something?

**Comment by dpksinghal** [comment]
:clock1:

File: Tasks/QuickPerfTest/CltTasksUtility.ps1 (hunk 1 above)
Comment (vinitsuri) [inline Tasks/QuickPerfTest/CltTasksUtility.ps1]: resolved

File: Tasks/QuickPerfTest/CltThresholdValidationHelper.ps1 (hunk 2 above)
Comment (vinitsuri) [inline Tasks/QuickPerfTest/CltThresholdValidationHelper.ps1]: resolved

**Pull request merged**